### ISSUE TITLE
add documentation!

### ISSUE THREAD

**Issue by mmckegg** [mmckegg/audio-slot#2]
This is a pretty cool module. TELL THE WORLD ABOUT IT!

**Comment by naugtur** [comment]
I found it on npm and I'd gladly find out what it can do without reading the code :)

**Comment by zya** [comment]
still don't get what this library is supposed to do. Although it seems like it should be doing something cool.
Documentation would really help me understand.

**Comment by mmckegg** [comment]
Haha, yeah. :kissing_cat: 

It is pretty much single handedly powering the audio engine in [Loop Drop](http://loopjs.com).

The reason I haven't written any docs is because it's a bit of a mess and needs to be split out into multiple modules. Also I kept changing the API every few weeks.

It allows you to declaratively describe a Web Audio API node setup using JSON, then push in a new state and then only update the AudioParams that have changed. Basically react for Web Audio.

**Comment by zya** [comment]
maybe a little example in the readme to keep people starting?
Like how you would make a simple oscillator -> gain -> destination example.

**Comment by yoshuawuyts** [comment]
Yeah, having at least a bit of an example would be much appreciated! (':

**Comment by mmckegg** [comment]
Gonna start documenting all this stuff next Tuesday. 

Promise! :blush: :ghost:

**Comment by yoshuawuyts** [comment]
:heart_eyes_cat:

**Comment by mmckegg** [comment]
Added a simple example to the readme. Also see https://github.com/mmckegg/audio-slot-param which has been extracted out of this module.
